dill/__diff.py

```diff
@@ -10,18 +10,12 @@
 Module to show if an object has changed since it was memorised
 """
 
-import os
-import sys
-import types
+import builtins, os, sys, types
```

Comment:
don't condense module imports to a single line

Reply:
Just googled about this. The "one module import by line" style is the PEP8 recommendation (although it doesn't give a rationale) and it seems to avoid merge conflicts in some cases, which is nice. Okay